scripts/zones/The_Ashu_Talif/mobs/Yazquhl.lua

```diff
@@ -0,0 +1,38 @@
+-----------------------------------
+-- Area: The Ashu Talif (Against All Odds)
+--  MOB: Yazquhl
+-----------------------------------
+require("scripts/globals/status")
+local ID = require("scripts/zones/The_Ashu_Talif/IDs")
+mixins = {require("scripts/mixins/job_special")}
+-----------------------------------
+
+function onMobSpawn(mob)
+    mob:addMod(dsp.mod.SLEEPRES, 150)
+    mob:addMod(dsp.mod.SILENCERES, 150)
+end
+
+function onMobFight(mob,target)
```

Comment:
this shouldn't go in onMobFight, or it'll add the listener over and over during combat.

you can use onMobWeaponskill() in a mob script the same way you use it in a mobskill script.  The core calls both.

Reply:
Ok i was copying the script from the black coffin captain its the same way but doesnt work